packages/api/drizzle/meta/_journal.json

```diff
@@ -78,6 +78,20 @@
       "when": 1726808187994,
       "tag": "0010_create_apps",
       "breakpoints": true
+    },
+    {
+      "idx": 11,
+      "version": "6",
+      "when": 1728457840317,
+      "tag": "0011_apps_external_id_unique",
```

Comment:
Hmm I wonder if this is a rebase issue, but I expect only one entry in the journal for this PR. Maybe we need to rebase and rerun the migration?

Reply:
On the main branch, the journal entry and snapshot for the 11th migration `0011_apps_external_id_unique.sql` are missing while the migration is present.